ui/address/utils/useAddressQuery.ts

```diff
@@ -35,6 +37,10 @@ export default function useAddressQuery({ hash, isEnabled = true }: Params): Add
       placeholderData: ADDRESS_INFO,
       refetchOnMount: false,
       retry: (failureCount, error) => {
+        if (error.status < 500) {
+          return false;
+        }
+
```

Comment:
I think we handle this case in the main `retry` function.
https://github.com/blockscout/frontend/blob/f39c931c0ebc5d0bdce46a3adb82c4175ac19644/lib/api/useQueryClientConfig.tsx#L10